src/SiteConfiguration.php

```diff
@@ -282,9 +342,16 @@ public function buildThemeSettingsForm(array &$form, FormStateInterface &$form_s
           $this->t('None'),
           $this->t('Footer'),
           $this->t('Header'),
+          $this->t('Homepage Layout'),
         ],
         '#description'    => $this->t('Check this box if you would like to display the "Be Boulder" slogan in the header.'),
       ];
+      $form['advanced']['ucb_homepage_header'] = [
```

Comment:
Can we get rid of the double-negative here, parsing this is confusing.  Also it's unchecked but not displaying by default.  We want it unchecked by default but displaying, currently this would wipe out the site title for every site even though it's not checked.